### ISSUE TITLE
Compliance Summary Page for Regulated Operations 

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-reporting#146]
Description: 
As a Regulated Operation I want a Compliance Summary page so that I can see final emissions totals. 

[WIREFRAME](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=17438-5696&t=wwVoNouq4uILpIVi-4)

**Question: what is initial_compliance_period?**
The spreadsheet says it has a value of 2024, but should that be pulled from the first year the facility reported rather than hardcoded?
Answer: Yes it will always be 2024. Hardcode it.

Acceptance Criteria: 
Given I am a Regulated Operation (Regulated, Opt-in or New Entrant)
When I have filled out all my required forms including emissions data, production data, allocation of emissions and any additional reporting data
Then I need to see a summary page with compliance information/totals (calculated from data input on previous forms). 
This is required as part of my compliance report, including my excess or credited emissions. 

Given I am a Developer
When I am building the Compliance Summary page
Then I need to reference the "Emissions Summary" spreadsheet saved in the SME teams channel under Files > Reporting App
in order to create the required calculations/totals. 

naics_code_regulatory_value model:
id pkey
naics_code fkey -> naics_code
reduction_factor decimal
tightening_rate decimal
valid_from timestamptz
valid_to timestamptz

product_emission_intensity model:
id pkey
product_id fkey -> product
product_weighted_average_emission_intensity decimal
valid_from timestamptz
valid_to timestamptz

#### Development Checklist:

- [ ] Compliance summary values are calculated using the parameters in the 'Emissions Summary' spreadsheet
- [ ] Regulatory values are defined by NAICS code as per the 'Registration fields pt 2' spreadsheet
- [x] A naics_code_regulatory_value model exists as described above to hold the regulatory values & valid to/from dates
- [x] A product_emission_intensity model exists as described above to hold the PWAEI values & valid to/from dates (values in Registration fields pt 2 spreadsheet)
- [ ] For each product the annual amount & PWAEI is shown (from database values)
- [ ] For each product, the allocated industrial process emissions & compliance emissions are calculated & shown
- [x] The compliance summary page has a continue & a back button (but no save button)
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Notes:

-

**Comment by rdromey** [comment]
@dleard to write AC and dev checklist

**Comment by patriciarussellCAS** [comment]
Blocked by #237

**Comment by patriciarussellCAS** [comment]
Note from Collab: Please add another data field under each Product: Production {April 1 - Dec 31 2024}

**Comment by patriciarussellCAS** [comment]
Based on my understanding, this page is working well. We will have UAT testers try and recalculate this to demonstrate that the calculations in the system are correct.
